js/src/modals/CreateAccount/store.js

```diff
@@ -234,20 +239,23 @@ export default class Store {
   createAccountFromGeth = (timestamp = Date.now()) => {
     return this._api.parity
       .importGethAccounts(this.gethAddresses.peek())
-      .then(() => {
-        return Promise.all(this.gethAddresses.map((address) => {
-          return this._api.parity.setAccountName(address, 'Geth Import');
-        }));
-      })
-      .then(() => {
-        return Promise.all(this.gethAddresses.map((address) => {
-          return this._api.parity.setAccountMeta(address, {
-            timestamp
+      .then((gethImported) => {
+        console.log('createAccountFromGeth', gethImported);
```

Comment:
On purpose - very specific "noise" in this case for an operation that happens once-off. So going to "ignore" the comment, but understand the concern very much for anything that is not basically a once-off :)